tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
> The last is the most consistent in terms of formatting […] the middle is what is used in mediawiki style.

I'm afraid I don't get the problem. Two of the three examples you gave are allowed when using the MediaWiki code style, including the one you refer to as "most consistent".

> […] I would prefer the first style because it is readable […]

Sorry, but no. This is as bad as a single-line `if ( … ) return …;`. Significant control structures like a `return …` should always be on their own line.

> As well as this case, for example: […]

Sorry, but I don't understand which one is causing problems and why.